res/css/views/auth/_AuthHeaderLogo.scss

```diff
@@ -22,10 +22,29 @@ limitations under the License.
 
 .mx_AuthHeaderLogo img {
     width: 100%;
+    width: 150px;
```

Comment:
There are 2 definitions for width here, please externalise a single value into the themes.  The values in the element themes will be 100% and our copy of the light theme in aether-host-files can specify 150px.  For the height value the value in the element themes, you will have to work out as based on https://developer.mozilla.org/en-US/docs/Web/CSS/height I'm not sure if previously having no value is equivalent to inherit, unset or auto.  In our copy of the light theme the value would be 100px

Reply:
to avoid confusion i have added this file in aether host files with the changes..